coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanUtils.scala

```diff
@@ -641,4 +641,17 @@ object LogicalPlanUtils extends StrictLogging {
       .distinct
       .map(_.toSeq)
   }
+
+  def getLogicalPlanTreeStringRepresentation(logicalPlan: LogicalPlan) : String = {
+    // iterate over all the children of the logical plan and get the string representation
+    val children = logicalPlan match {
```

Comment:
Nice dfs algorithm.
I guess
planTreeStringRepresentationOfChildren
is a more clear name?